src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline

```diff
@@ -1650,6 +1652,7 @@ import java.security.GeneralSecurityException;
 public class AdExchangeBuyerExample {
   public void run() throws IOException, GeneralSecurityException {
     // TODO: Assign values to the desired fields of `requestBody`.
+    // Only the assigned fields will be changed.
```

Comment:
Just curious: do you know whether every time that the HTTP verb is `PATCH`, the client library method is also `.patch(..)`? Looking at these baselines, it seems to be, which is good.